lib/renderer.coffee

```diff
@@ -3,8 +3,16 @@
 path = require 'path'
 fs = require 'fs-plus'
 cheerio = require 'cheerio'
+
 # No direct dependence with Highlight because it requires a compilation. See #63 and #150 and atom/highlights#36.
-Highlights = require path.join atom.packages.resolvePackagePath('markdown-preview'), '..', 'highlights'
+markdownPreviewPath = atom.packages.resolvePackagePath 'markdown-preview'
+commonHighlightsPath = path.join markdownPreviewPath, '..', 'highlights'
+if fs.isDirectorySync commonHighlightsPath
+  Highlights = require commonHighlightsPath
+else
+  # Fix Fedora specific problem #226
+  Highlights = require path.join markdownPreviewPath, 'node_modules', 'highlights'
```

Comment:
I just did a fresh install of Atom 1.15.0 (atom.x86_64.rpm) on Fedora 21 (yes, I realize I'm using a very old version of Fedora). The latest AsciiDoc Preview release finds the markdown-preview package without a problem. I can browse in the JavaScript debugger and see the markdown-preview package under /usr/share/atom/resources (which appears to be some sort of virtual file system). If this is a problem, it doesn't affect Fedora 21.

However, while testing, what I discovered is that the colors are no longer applied. It probably has something to do with this message:

> Starting from Atom v1.13.0, the contents of atom-text-editor elements are no longer encapsulated within a shadow DOM boundary. This means you should stop using :host and ::shadow pseudo-selectors, and prepend all your syntax selectors with syntax--.